File: RadarChartView/src/main/java/com/ma/customview/MainActivity.java
package com.ma.customview;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity implements View.OnClickListener{

    private TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        initView();
    }

    private void initView() {
        textView=findViewById(R.id.main_cobweb);
        textView.setOnClickListener(this);
    }

    @Override
    public void onClick(View view) {
        switch (view.getId()){
            case R.id.main_cobweb:
                Intent toCobweb=new Intent(this,CobwebActivity.class);
                startActivity(toCobweb);
                break;
        }
    }
}

File: app/src/main/java/com/ma/customview/MainActivity.kt
package com.ma.customview

import android.content.Intent
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : BaseActivity(), View.OnClickListener {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initView()
    }

    private fun initView() {
        main_radarchat.setOnClickListener(this)
        main_googlesearch.setOnClickListener(this)
    }

    override fun onClick(view: View) {
        when (view.id) {
            R.id.main_radarchat -> {
                val toRadarChat = Intent(this, RadarChartActivity::class.java)
                startActivity(toRadarChat)
            }
            R.id.main_googlesearch -> {
                val toGoogleSearch = Intent(this, GoogleSearchActivity::class.java)
                startActivity(toGoogleSearch)
            }
        }
    }
}